src/pages/[guild]/events.tsx

```diff
@@ -133,39 +135,40 @@ const GuildEvents = (): JSX.Element => {
           <TabButton href={`/${urlName}/activity`}>Activity log</TabButton>
         )}
       </Tabs>
-      {!data && !error && isLoading && (
-        <FallbackFrame isLoading text="Searching for events..." />
-      )}
-      {!data && !error && !isLoading && (
-        <FallbackFrame
-          icon={NoteBlank}
-          title="No events yet"
-          text="Your guild has no upcoming event currently"
-        />
-      )}
-
-      {!isLoading && !!error && (
-        <FallbackFrame
-          icon={WarningOctagon}
-          text="Something went wrong, couldn't load events."
-        />
-      )}
-
-      {!isLoading && !error && data?.length === 0 && (
+      {isLoading ||
```

Comment:
I think this is why there's no loading state on initial load? 👀 I think it should be `(isLoading || data === undefined) && ...`?